xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -2072,6 +2072,16 @@
     "params": [],
     "returns": "string"
   },
+  "System.InhibitIdleShutdown": {
+    "type": "method",
+    "description": "Toggle the inhibit idle shutdown timer",
```

Comment:
This description seems wrong. The functionality either inhibits or stops inhibiting the idle shutdown timer if I understood correctly.